rxbinding-design/src/androidTest/java/com/jakewharton/rxbinding/support/design/widget/RxFloatingActionButtonTest.java

```diff
@@ -0,0 +1,33 @@
+package com.jakewharton.rxbinding.support.design.widget;
+
+import android.content.Context;
+import android.support.design.widget.FloatingActionButton;
+import android.support.test.InstrumentationRegistry;
+import android.support.test.annotation.UiThreadTest;
+import android.support.test.runner.AndroidJUnit4;
+import android.view.View;
+
+import org.junit.Test;
+import org.junit.runner.RunWith;
+
+import io.reactivex.functions.Consumer;
+
+import static org.junit.Assert.assertEquals;
+
+@RunWith(AndroidJUnit4.class)
+public class RxFloatingActionButtonTest {
+
+  private final Context context = InstrumentationRegistry.getContext();
+  private final FloatingActionButton fab = new FloatingActionButton(context);
+
+  @Test
+  @UiThreadTest
+  public void visibility() throws Exception {
```

Comment:
Travis CI build had failed on this test case. The reason is related to used by default theme and Gradle says that `You need to use a Theme.AppCompat theme (or descendant) with the design library.`

Check: https://travis-ci.org/JakeWharton/RxBinding/jobs/294786206